### ISSUE TITLE
LDAP Admins Unable to see details/flows/host/etc...

### ISSUE THREAD

**Issue by hytek3000** [ntop/ntopng#1785]
Any LDAP enabled admin, when they log in they are unable to see flows of any kind, or pretty much anything besides summary pages and graphs.

LDAP Settings:
LDAP Search Path: (bind path to OU where admin accounts are located)
LDAP User Group: (blank as we are not using unprivileged users)
LDAP Admin Group: (bind path to domain admin group)

What ldap admins can view:
1. Dashboard
2. Traffic Report
3. Historical Summary results only (no flow data or talkers)
4. Interface home/packets/protocol/graph summary (no flow data or talkers or peers)
5. Preferences

What ldap admins can NOT view:
1. Any flows, talkers, or peers from any page
2. Any details for host from any page
3. Talkers, Flows, Host, Networks, pretty much everything else

Version | 3.4.180516 -  Enterprise Edition
-- | --
Built on | CentOS Linux release 7.4.1708 (Core)
-- | --

We are using MariaDB to store flows as well. Nothing shows up in the log file as an error either.

Thanks.

**Comment by emanuele-f** [comment]
Hello, I cannot reproduce the issue. Have you selected the proper interface from the interfaces menu? If you connect with local admin account, can you see that info? Can you please post screenshots and your ntopng configuration used?

**Comment by hytek3000** [comment]
This issue is considered resolved, but it required me to restart redis-server and NTOPNG for me to see flows under an LDAP admin account.

Steps taken from beginning to end:
1. Setup ntopng, nprobe, mariadb, and tested everything was working with local admin (working fine)
2. Enabled LDAP/LOCAL accounts
3. Logged in with LDAP admin account; was able to see summaries, but unable to see details, flows, host, or history
4. Restarted NTOPNG (stop ntopng; start ntopng); same issue
5. Restarted both ntopng and redis-server (stop ntopng; stop redis-server;  start redis-server; start ntopng) and I am now able to see all flows, host, details and everything the local admin can see

(note: the linux commands were shortened in this post: used systemctl proper to control the services)

I had to force redis-server to restart before I was able to see everything with ldap admin.

Thanks.
